### DISCUSSION TITLE
Any roadmap for the next rustdesk release

### DISCUSSION THREAD

**Discussion by mikekgr** [rustdesk/rustdesk#884] [General]
Hello dear developers.
Do you have any idea when next release could be happen?
I think you have ironing out quite a few bugs so it's worth it.
Thank

**Comment by rustdesk** [comment]
Waiting for https://github.com/rustdesk/rustdesk/issues/406

**Reply by mikekgr** [reply]
@rustdesk OK Thanks keep your good work in rustdesk
